pulsar-client/src/main/java/org/apache/pulsar/client/impl/ReaderBuilderImpl.java

```diff
@@ -238,7 +238,7 @@ public ReaderBuilder<T> autoUpdatePartitions(boolean autoUpdate) {
     }
 
     @Override
-    public ReaderBuilder<T> autoUpdatePartitionsInterval(int interval, TimeUnit unit) {
```

Comment:
This will introduce a binary compatibility issue between different versions of the client